documenters_aggregator/spiders/ccc.py

```diff
@@ -0,0 +1,139 @@
+# -*- coding: utf-8 -*-
+"""
+All spiders should yield data shaped according to the Open Civic Data
+specification (http://docs.opencivicdata.org/en/latest/data/event.html).
+"""
+import scrapy
+import re
+
+from datetime import datetime
+from pytz import timezone
+
+
+class CccSpider(scrapy.Spider):
+    name = 'ccc'
+    allowed_domains = ['http://www.ccc.edu/departments/Pages/Board-of-Trustees.aspx']
+    start_urls = ['http://http://www.ccc.edu/departments/Pages/Board-of-Trustees.aspx']
```

Comment:
an extra http:// here

Reply:
Maybe try using this site for start_urls instead? http://www.ccc.edu/events/Pages/default.aspx?dept=Board%20of%20Trustees